ampl-converter/src/main/java/com/powsybl/ampl/converter/AmplNetworkReader.java

```diff
@@ -279,9 +296,13 @@ private Void readShunt(String[] tokens) {
         }
 
         if (sc.getModelType() == ShuntCompensatorModelType.NON_LINEAR) {
-            throw new PowsyblException("non linear shunt not supported yet");
+            // nothing.
+            if (sections == 0) {
+                sc.setSectionCount(sections);
+            } // else nothing.
```

Comment:
Are the `// nothing` really useful?

Reply:
:-) I have put a TODO instead. I need help to continue with non linear shunts!